server/src/main/java/com/linecorp/centraldogma/server/internal/admin/authentication/User.java

```diff
@@ -33,18 +33,34 @@
 
     private static final long serialVersionUID = -5429782019985526549L;
 
-    public static final String ADMIN = "ROLE_ADMIN";
-    public static final String USER = "ROLE_USER";
+    // System-wide roles for a user. It is different from the role in a project.
+    public static final List<String> USER_ROLE = ImmutableList.of("ROLE_USER");
```

Comment:
I'm thinking of changing these `ROLE_*` to `TIER_*`. WDYT? @minwoox @trustin Is there any better naming?